torch/nn/attention/varlen.py

```diff
@@ -33,8 +33,7 @@ class AuxRequest(NamedTuple):
     lse: bool = False
 
 
-# import failures when I try to register as custom op
-# @torch.library.custom_op("torch_nn_attention::_varlen_attn", mutates_args={})
+@torch.library.custom_op("torch_nn_attention::_varlen_attn", mutates_args={})
```

Comment:
lets make the lib name better, torch:: ? 

@zou3519 Is there any naming convention for python ops in PT?

Reply:
@liangel-02 lets just change to `torch_attn`